sdk/python/kfp/dsl/schemas/classification_metrics.yaml

```diff
@@ -0,0 +1,145 @@
+title: ClassificationEvaluationMetrics
```

Comment:
I don't see these yaml files being referenced in code, but when you do need these, they should be listed in https://github.com/kubeflow/pipelines/blob/master/sdk/python/MANIFEST.in